### ISSUE TITLE
Fatal error starting autokey <built-in function connectSlotsByName> returned a result with an error set

### ISSUE THREAD

**Issue by sicherist** [autokey/autokey#318]
## Classification:

Bug

## Reproducibility:

Always

## Version

AutoKey version: 0.95.7-0

Used GUI (Gtk, Qt, or both): QT

Installed via: (PPA, pip3, …). pre built .deb package

Linux Distribution: Kubuntu 19.10

## Summary

After upgrading from Kubuntu 19.04 to 19.10, autokey-qt starts with an error message "Fatal error starting autokey <built-in function connectSlotsByName> returned a result with an error set". I can still use autokey via hotkeys to paste my phrases but I can not access the main window. When I right-click on the autokey tray-icon it takes very long to open the context-menu. 

## Steps to Reproduce (if applicable)
Install and start qutokey-qt on Kubuntu 19.10 (according to[ this thread](https://groups.google.com/forum/#!topic/autokey-users/FhDnMVUJ59Y) it looks like other distributions with KDE 5.16 could be affected too.)

## Expected Results

No error message is expected.

## Actual Results

A small window with an error message opens "Fatal error starting autokey <built-in function connectSlotsByName> returned a result with an error set"

Starting "autokey-qt -l" from the terminal gives this log:

```
2019-11-05 09:03:05,345 INFO - root - Initialising application
2019-11-05 09:03:05,375 INFO - root - Initialise global hotkeys
2019-11-05 09:03:05,375 INFO - config-manager - Loading config from existing file: /home/user/.config/autokey/autokey.json
2019-11-05 09:03:05,376 DEBUG - config-manager - Loading folder at '/home/user/.config/autokey/data/My Phrases'
2019-11-05 09:03:05,379 DEBUG - config-manager - Loading folder at '/home/user/.config/autokey/data/Sample Scripts'
2019-11-05 09:03:05,380 INFO - config-manager - Configuration changed - rebuilding in-memory structures
2019-11-05 09:03:05,380 DEBUG - inotify - Adding watch for /home/user/.config/autokey/data/My Phrases
2019-11-05 09:03:05,380 DEBUG - inotify - Adding watch for /home/user/.config/autokey/data/Sample Scripts
2019-11-05 09:03:05,380 INFO - config-manager - Successfully loaded configuration
2019-11-05 09:03:05,380 DEBUG - inotify - Adding watch for /home/user/.config/autokey/data
2019-11-05 09:03:05,380 DEBUG - inotify - Adding watch for /home/user/.config/autokey
2019-11-05 09:03:05,380 DEBUG - config-manager - Global settings: {'isFirstRun': False, 'serviceRunning': True, 'menuTakesFocus': False, 'showTrayIcon': True, 'sortByUsageCount': False, 'promptToSave': False, 'enableQT4Workaround': False, 'interfaceType': 'XRecord', 'undoUsingBackspace': True, 'windowDefaultSize': [922, 716], 'hPanePosition': 260, 'columnWidths': [151, 52, 114], 'showToolbar': True, 'notificationIcon': 'autokey-status', 'workAroundApps': '.*VirtualBox.*|krdc.Krdc', 'triggerItemByInitial': False, 'scriptGlobals': {}}
2019-11-05 09:03:05,380 INFO - service - Starting service
2019-11-05 09:03:05,407 DEBUG - interface - Enabling sending using Alt-Grid
2019-11-05 09:03:05,407 DEBUG - interface - Modifier masks: {<Key.SHIFT: '<shift>'>: 1, <Key.CONTROL: '<ctrl>'>: 4, <Key.ALT: '<alt>'>: 8, <Key.ALT_GR: '<alt_gr>'>: 128, <Key.SUPER: '<super>'>: 64, <Key.HYPER: '<hyper>'>: 64, <Key.META: '<meta>'>: 8, <Key.NUMLOCK: '<numlock>'>: 16}
2019-11-05 09:03:05,437 DEBUG - interface - Alt-Grid: XK_ISO_Level3_Shift, 65027
2019-11-05 09:03:05,437 DEBUG - interface - X Server Keymap, listing unmapped keys.
2019-11-05 09:03:05,437 DEBUG - interface - No mapping for [`]
2019-11-05 09:03:05,437 DEBUG - interface - No mapping for [^]
2019-11-05 09:03:05,437 DEBUG - iomediator - Set modifier Key.CAPSLOCK to False
2019-11-05 09:03:05,437 DEBUG - iomediator - Set modifier Key.NUMLOCK to True
2019-11-05 09:03:05,438 DEBUG - interface - Grabbing hotkey: ['<super>'] 'k'
2019-11-05 09:03:05,438 DEBUG - interface - Grabbing hotkey: ['<shift>', '<super>'] 'k'
2019-11-05 09:03:05,438 DEBUG - interface - Grabbing hotkey: ['<ctrl>', '<shift>', '<super>'] '<'
2019-11-05 09:03:05,439 DEBUG - interface - __flushEvents: Entering event loop.
2019-11-05 09:03:05,439 INFO - iomediator - Created IoMediator instance, current interface is: <XRecordInterface(XInterface-thread, initial daemon)>
2019-11-05 09:03:05,443 INFO - interface - XRecord interface thread starting
2019-11-05 09:03:05,444 INFO - service - Service now marked as running
2019-11-05 09:03:05,444 DEBUG - root.Qt-GUI.System-tray-notifier - Creating system tray icon notifier.
2019-11-05 09:03:05,447 INFO - root.Qt-GUI.System-tray-notifier - Creating static context menu actions.
2019-11-05 09:03:05,448 DEBUG - root.Qt-GUI.System-tray-notifier - Show tray icon enabled in settings: True
2019-11-05 09:03:05,448 INFO - root.Qt-GUI.System-tray-notifier - Rebuilding model item actions, adding all items marked for access through the tray icon.
2019-11-05 09:03:05,448 DEBUG - phrase-menu - Sorting phrase menu by item name/title
2019-11-05 09:03:05,448 DEBUG - phrase-menu - Sorting phrase menu by item name/title
2019-11-05 09:03:05,452 DEBUG - root.Qt-GUI.System-tray-notifier - About to show the tray icon.
2019-11-05 09:03:05,480 INFO - root.Qt-GUI.System-tray-notifier - System tray icon notifier created.
2019-11-05 09:03:05,480 DEBUG - root.Qt-GUI.CentralWidget - CentralWidget instance created.
2019-11-05 09:03:05,632 ERROR - root - Fatal error starting AutoKey: <built-in function connectSlotsByName> returned a result with an error set
AttributeError: __getattribute__
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/autokey/qtapp.py", line 132, in __init__
    self.configWindow = ConfigWindow(self)
  File "/usr/lib/python3/dist-packages/autokey/qtui/configwindow.py", line 44, in __init__
    self.setupUi(self)
  File "<string>", line 12, in setupUi
  File "/usr/lib/python3/dist-packages/autokey/qtui/centralwidget.py", line 40, in __init__
    self.setupUi(self)
  File "<string>", line 38, in setupUi
  File "/usr/lib/python3/dist-packages/autokey/qtui/phrasepage.py", line 34, in __init__
    self.setupUi(self)
  File "<string>", line 46, in setupUi
  File "/usr/lib/python3/dist-packages/autokey/qtui/settingswidget.py", line 40, in __init__
    self.hotkey_settings_dialog = HotkeySettingsDialog(self)
  File "/usr/lib/python3/dist-packages/autokey/qtui/dialogs/hotkeysettings.py", line 49, in __init__
    self.setupUi(self)
  File "<string>", line 58, in setupUi
SystemError: <built-in function connectSlotsByName> returned a result with an error set
```

**Comment by josephj11** [comment]
Until this gets addressed, you may want to try using the GTK front end. Aside from pulling in a bunch of libraries as dependencies (which you may already have installed if you have other GTK applications), it will perform almost identically to the QT version.

**Comment by sicherist** [comment]
Thanks for the workaround!

**Comment by luziferius** [comment]
@sicherist @josephj11 Fixed the issue in [v0.95.8](https://github.com/autokey/autokey/releases/tag/v0.95.8)

**Comment by sicherist** [comment]
Thank you!
